packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -142,20 +146,51 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
         return "";
     }
 
-    public async createBlob(file: Uint8Array): Promise<api.ICreateBlobResponse> {
-        this.checkSnapshotUrl();
+    public async createBlob(file: IsoBuffer): Promise<api.ICreateBlobResponse> {
+        this.checkAttachmentPOSTUrl();
 
-        return PerformanceEvent.timedExecAsync(
-            this.logger,
-            {
-                eventName: "createBlob",
-                size: file.length,
-            },
-            async () => {
-                // Future implementation goes here
-                // Need to wrap implementation with getWithRetryForTokenRefresh()
-                throw new Error("StandardDocumentStorageManager.createBlob() not implemented");
-            });
+        const response = await getWithRetryForTokenRefresh(async (options) => {
+            const storageToken = await this.getStorageToken(options, "CreateBlob");
+            const { url, headers } = getUrlAndHeadersWithAuth(`${this.attachmentPOSTUrl}/content`, storageToken);
+            headers["Content-Type"] = "application/octet-stream";
+
+            return PerformanceEvent.timedExecAsync(
+                this.logger,
+                {
+                    eventName: "createBlob",
+                    size: file.length,
+                },
+                async () => fetchHelper<api.ICreateBlobResponse>(
+                    url,
+                    {
+                        body: file,
```

Comment:
file is a buffer, not a string, right? I believe you want to stringify.
Which likely means using base64 encoding.

Reply:
file is a Buffer, but the content-type is "octet-stream". base64 shouldn't be necessary

Reply:
You are right, there is support for wider range of types here, but are you sure IsoBuffer implements any of the supported / expected things by fetch()? from documentaiton:
https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch

Any body that you want to add to your request: this can be a Blob, BufferSource, FormData, URLSearchParams, USVString, or ReadableStream object.

Reply:
Also does SPO accept binary data? Is that what API expects and deals with? Just want to make it's the case

Reply:
I've changed it from `IsoBuffer` to `Uint8Array` due to the reasons you gave below, but I believe both `IsoBuffer` and `Uint8Array` are fine here since `Uint8Array` is a `TypedArray` and therefore a `BufferSource`:

https://developer.mozilla.org/en-US/docs/Web/API/BufferSource
>BufferSource is a typedef used to represent objects that are either themselves an ArrayBuffer, or which are a TypedArray providing an ArrayBufferView.

 and `IsoBuffer` extends `Uint8Array`.